docs/new-audits.md

```diff
@@ -8,13 +8,33 @@ Lighthouse audits that surface in the report should be:
 - Contribute significantly towards making the mobile web experience better for end users. 
 - Not have a significant impact on our runtime performance or bundle size. 
 - Something that is new, and not something that is already measured by existing audits. 
-- Important for our strategic goals as a product.
```

Comment:
This is another way to say "we have veto control on what lands in core", which goes without saying. low-value to have here.